README.md

```diff
@@ -1 +1,127 @@
+[Pop up window]
+<img width="703" alt="Screen Shot 2022-10-06 at 16 28 38" src="https://user-images.githubusercontent.com/92399469/194403937-74d63dcc-18e3-490b-9e9e-b8815df5f759.png">
+[Vehicles screen]
+<img width="703" alt="Screen Shot 2022-10-06 at 16 29 18" src="https://user-images.githubusercontent.com/92399469/194403991-59693a2c-0faf-4f18-8554-777e9040f3c6.png">
+[Add vehicles screen]
+<img width="703" alt="Screen Shot 2022-10-06 at 16 29 39" src="https://user-images.githubusercontent.com/92399469/194404064-b89327f4-7ed3-4cb4-a2e3-8a3d8128eef8.png">
+[Delete vehicle screen]
+<img width="699" alt="Screen Shot 2022-10-06 at 16 29 56" src="https://user-images.githubusercontent.com/92399469/194404122-69540a18-d4cb-4496-888f-f8b1c3e28fbc.png">
+[Reserve a vehicle screen]
+<img width="700" alt="Screen Shot 2022-10-06 at 16 30 12" src="https://user-images.githubusercontent.com/92399469/194404172-d21c1fb3-2c3a-4614-b93c-2dc763638996.png">
+[My reservation screen]
+
+
+## Live Demo
+
+- [Live Demo Link]()
+
+## Link to Back-end repo
+- https://github.com/Fatima-hub333/appointment-diary-backend
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+- Copy this link `https://github.com/Fatima-hub333/appointment-diary-front-end.git`.
+- Get the directory that you want to clone the repository.
+- Open the command prompt in this directory.
+- Write `git clone https://github.com/Fatima-hub333/appointment-diary-front-end.git`.
+- Go to the repository folder in your command prompt, run `cd 
+appointment-diary-front-end
+`.
+- Run `npm install`
+
+### Prerequisites
+
+1. NodeJS
+2. NPM
+
+### Build
+
+Type these commands into the terminal:
+
+- npm run build
+- Open dist/index.html in your web browser.
+
+### Run the application in local environment
+
+Type this command in your terminal
+
+- npm run start
+- Your web browser will open on localhost
+
+### Run tests
+
+Type this command into the terminal:
+
+- npm run test
+
+## Authors
+
+👤 **Bassem Shams**
+
+- GitHub: [@basem909](https://github.com/basem909)
+- Twitter: [@ShamsBassem](https://twitter.com/ShamsBassem)
+- LinkedIn: [Bassem Abdelrahman](https://www.linkedin.com/in/bassem-shams-126a29134/)
+
+👤 **Isaac maqueen**
+
+- GitHub: [@kinginthenorthcodez](https://github.com/kinginthenorthcodez)
+- Twitter: [@kinginthenorthcodez](https://twitter.com/kinginthenorthcodez)
+
+👤 **Fatima Zahir**
+
+- GitHub: @Fatima-hub333
+- Twitter: @twitterhandle
+- LinkedIn: LinkedIn
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Original design idea by Murat Korkmaz on Behance.](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) 
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.

```

Comment:
- [ ] Nice work adding a license file to your project.  Please, kindly correct the link on your readme file because it does not work.